packages/wagtail/modules/services/Wagtail/index.ts

```
@@ -0,0 +1,7 @@
+import axios from '../axios'
+
+export class WagtailAPI {
+  static baseURL = `${process.env.NEXT_PUBLIC_WAGTAIL_API_BASE_URL}`
+
+  static axios = axios
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)